teachopencadd/talktorials/002_compound_adme/talktorial.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<p>adapt fct name, if you change it according to this review </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>